tests/test_core_subset.py

```diff
@@ -727,13 +727,12 @@ def test_wraps(self, tmp_netcdf_filename, nimbus, clisops_test_data):
             float(np.mean(subtas.isel(time=0))), 281.092, 3
         )
 
-        assert sub.crs.prime_meridian_name == "Greenwich"
-        assert sub.crs.grid_mapping_name == "latitude_longitude"
```

Comment:
What would be the alternative approach for ensuring that this information remains available?

Reply:
I think the point if it was already available it will stay, but we were adding it based on a guess that was sometimes wrong.

Reply:
Can we find a more reliable way of ensuring this information is available if needed? If we don't want to rely on guessing, could we add something to a call signature that forces this info to be added?

Reply:
I'm not sure I see why the `subset_shape` function would need to use or handle the grid mapping information ?

The way it is currently written, longitude and latitude coordinates are required to exist and are the only spatial information needed, no matter what actual projection the data is using. The only question remaining is the `wrap_lon` stuff.